packages/accounts/src/msca/plugins/multisig/signer.ts

```diff
@@ -0,0 +1,106 @@
+import type {
+  Address,
+  BundlerClient,
+  SmartAccountSigner,
+} from "@alchemy/aa-core";
+import {
+  hashMessage,
+  hashTypedData,
+  type Hash,
+  type Hex,
+  type SignableMessage,
+  type Transport,
+  type TypedData,
+  type TypedDataDefinition,
+} from "viem";
+import { MultisigPlugin, MultisigPluginAbi } from "./plugin.js";
+
+type MultisigMessageSignerParams<
+  TTransport extends Transport,
+  TSigner extends SmartAccountSigner
+> = {
+  client: BundlerClient<TTransport>;
+  accountAddress: Address;
+  signer: () => TSigner;
+  threshold: bigint;
+  pluginAddress?: Address;
+};
+
+export const multisigMessageSigner = <
```

Comment:
can we call this `multisigSignMethods` since it's a bit confusing that it's not really a signer